tests/async/test_dynamic_browser_name.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from playwright.sync_api import sync_playwright
+
+
+def test_dynamic_browser_name(browser_name, launch_arguments):
+    with sync_playwright() as p:
+        browser = p[browser_name].launch(**launch_arguments)
+        page = browser.new_page()
+        page.goto("https://github.com")
```

Comment:
We don't want to rely on external websites for the test. A simple assert page.evaluate("1 + 1") == 2 would be enough.

Reply:
Fixed